Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -381,8 +381,7 @@ public static bool SendToReady(NetworkIdentity identity, short msgType, MessageB
                 foreach (KeyValuePair<int, NetworkConnection> kvp in identity.observers)
                 {
                     bool isSelf = kvp.Value == identity.connectionToClient;
-                    if ((!isSelf || includeSelf) &&
-                        kvp.Value.isReady)
+                    if ((!isSelf || includeSelf || kvp.Value is ULocalConnectionToClient) && kvp.Value.isReady)
```

Comment:
Is this correct?

What happens if includeSelf == false and player is in host mode.   With your code it would send it to host,  but it should not.

Reply:
Yes it should send to host...that's the point of the change.